docs/END_USER_DOCUMENT.md

```diff
@@ -534,8 +546,20 @@ The table above includes only synchronous clients, and the code samples in this
 
 All Mocca classes mentioned in the table above work as a wrapper containing a default instance of the HTTP client. If the application needs the HTTP client to be configured in a certain manner, there are two ways to achieve that:
 
-1. Mocca client builder could be used to do so, providing a common API regardless of the type of HTTP client used.
-1. A custom instance of the HTTP client could be provided as a constructor parameter to the Mocca HTTP client wrapper.
+1. Mocca client builder could be used to do so, providing a common API regardless of the type of HTTP client used. All currently supported Mocca HTTP clients allow this type of configuration, except the *JAX-RS 2 client*.
+1. A custom instance of the HTTP client (already configured as desired) could be provided as a constructor parameter to the Mocca HTTP client wrapper. This is currently the only way to customize the HTTP client configuration when using a *JAX-RS 2 client*.
+
+In the example below, Mocca client builder API (via method `options`) is used to configure the client connection and read timeout, and whether it should follow redirects or not.
+
+``` java
+BooksAppClient client = MoccaClient.Builder
+    .sync("http://localhost:8080/booksapp")
+    .client(new MoccaOkHttpClient())
+    .options(1000, 1000, false)
+    .build(BooksAppClient.class);
+```
```

Comment:
Depending on the answer to the question in my previous comment, this example should be removed.